tests/NLog.UnitTests/GetLoggerTests.cs

```diff
@@ -121,43 +121,32 @@ public void GenericGetCurrentClassLoggerTest()
 
         public class InvalidLogger
         {
-           private InvalidLogger()
-           {
-	       }
+            private InvalidLogger()
+            {
+            }
         }
-        
+
+
         [Fact]
         public void InvalidLoggerConfiguration_ThrowsConfigurationException_IfThrowExceptionsFlagIsSet()
         {
-            bool ExceptionThrown = false;
-            try 
+            Assert.Throws<NLogConfigurationException>(() =>
             {
                 LogManager.ThrowExceptions = false;
                 LogManager.GetCurrentClassLogger(typeof(InvalidLogger));
-            }
-            catch (NLogConfigurationException)
-            {
-                ExceptionThrown = true;
-            }
-            Assert.True(ExceptionThrown);
+            });
+
         }
-        
+
         [Fact]
         public void InvalidLoggerConfiguration_ThrowsConfigurationException_IfThrowExceptionsFlagIsNotSet()
         {
-            bool ExceptionThrown = false;
-            try 
+            Assert.Throws<NLogConfigurationException>(() =>
             {
                 LogManager.ThrowExceptions = true;
```

Comment:
I see many tests restore this value in a `finally` block, and I believe it is good practice.